app/containers/UIKit/MultiSelect/index.tsx

```diff
@@ -22,8 +22,8 @@ interface IMultiSelect {
 	context?: number;
 	loading?: boolean;
 	multiselect?: boolean;
-	onSearch: Function;
-	onClose: Function;
+	onSearch?: Function;
+	onClose?: Function;
```

Comment:
Can we send how the function is composed instead of setting `Function`? I think `Function` is too ambiguous and open to anything